### ISSUE TITLE
Safety of subclassing

### ISSUE THREAD

**Issue by abeverley** [rjbs/HTML-FromText#5]
I note that in the documentation it states regarding subclassing that: "At the time of this release, the internals of HTML::FromText are in a state of development and cannot be expected to stay the same from release to release. I expect that release version 3.00 will be analogous to a 1.00 release of other software. This is because the current maintainer has rewritten this distribution from the ground up for the 2.x series."

Given that the there has not been a release of the module for several years, I wondered whether this is still applicable?

The reason is that I want to add ```rel=nofollow``` to generated links. I'd prefer to use native module functionality rather than subclassing, but would welcome any advice. Would some sponsored development be appropriate here?

Thanks,

Andy

**Comment by rjbs** [comment]
I… have no recollection of this library even existing.  It looks like I got involved very briefly, nine years ago, and never thought about it since then.

I'm not actually sure what you're asking about, but almost certainly the answer is, "I will hand over this module to some other maintainer, because I have no use for it."  Sorry!

**Comment by abeverley** [comment]
Thanks @rjbs - love the reply! I will see if I can find someone to take over the module and add the functionality or at least update the docs as appropriate.

Just to note here for the purposes of this issue, I would also like the ability to add ```target="_blank"``` to URLs.

**Comment by domm** [comment]
Hey @rjbs, @abeverley asked me to take over maintainance of this module (and add the requested feature). If it's OK to you, could you transfer PAUSE ownership to DOMM (or start with setting up comaint)? I've already cloned this repo, and can let a future CPAN release point to my fork. Hm, it would of course also make sense to somehow migrate the issues / pull-reqs, though I guess that could be done by hand

Greetings,
domm

**Comment by domm** [comment]
I think we can transfer issues if the user has admin rights to both repos. I've invited @rjbs to my fork, so he can move the issues. Or we can do it the other way round :-)

**Comment by rjbs** [comment]
Oh no, I never followed through and this has expired.   I'm afraid that when this happened, I was abroad and distracted.  If you'd still like, we can make this happen now!

**Comment by abeverley** [comment]
Thanks, I'm still keen if @domm is still able to do so :)

**Comment by domm** [comment]
@rjbs @abeverley yeah, sure, esp if the plan still is to just hand owner/comaint over to me, than I can push "my" fork to CPAN etc. But if @rjbs wants to keep maintaining that's of course also OK :-)

**Comment by rjbs** [comment]
Nope, I would like to step away from this.  I will happily destroy this repo!  To transfer the issues, I believe I need to be a collaborator on your fork, @domm. You invited me, but it expired.  If you invite again, I'll be quicker.

**Comment by rjbs** [comment]
I have accepted the invite, but can't transfer the issues.  Searching around, I can't tell.  Either I need more permission on domm's repo … or I can't.  Meanwhile, I will give domm permission on this one.

**Comment by domm** [comment]
"Adding collaborators" is the only option I can find to give you access. But I guess the github way is to just use my fork as the basis for new CPAN release...
